packages/web-react/src/components/Card/demo/CardMediaOptions.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { Button } from '../../Button';
+import { Grid, GridItem } from '../../Grid';
+import Card from '../Card';
+import CardBody from '../CardBody';
+import CardEyebrow from '../CardEyebrow';
+import CardFooter from '../CardFooter';
+import CardLink from '../CardLink';
+import CardMedia from '../CardMedia';
+import CardTitle from '../CardTitle';
+import { MEDIA_IMAGE } from './constants';
+
+// https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors
+
+export const CardMediaOptions = () => (
+  <Grid cols={{ mobile: 1, tablet: 2, desktop: 4 }} UNSAFE_style={{ gridAutoFlow: 'dense' }}>
+    <GridItem UNSAFE_className="d-grid" UNSAFE_style={{ ['--grid-item-row-end-desktop' as string]: 'span 2' }}>
```

Comment:
i think we have props for row end and so on. You shouldnt need to use unsafe style for griditem